src/lib/ab-testing/config.ts

```diff
@@ -0,0 +1,39 @@
+import { ABTestConfig } from "./types"
+
+function getEnvVar(key: string, defaultValue: string = ""): string {
+  return process.env[key] || defaultValue
```

Comment:
this could be another potential issue where we store dynamic variables instead of static ones. This could lead to missing vars if the bundler can't statically analyze the vars.

Reply:
gone!